packages/fxa-settings/src/lib/auth-errors/en.ftl

```diff
@@ -12,14 +12,21 @@ auth-error-114-generic = You’ve tried too many times. Please try again later.
 #   $retryAfter (String) - Time required before retrying a request. The variable is localized by our
 #                          formatting library (momentjs) as a "time from now" and automatically includes
 #                          the prefix as required by the current locale (for example, "in 15 minutes", "dans 15 minutes").
+auth-error-110 = Invalid Token
```

Comment:
Remove? We have this one on L6.

Reply:
Ideally this one wouldn't be user-facing 🤔

Reply:
Agreed, since it's already there though, and it can be used in multiple scenarios, I'm a bit hesitant to remove it entirely.